ui/enums.js

```diff
@@ -47,4 +47,5 @@ shaka.ui.Enums.MaterialDesignIcons = {
   'RECENTER_VR': 'control_camera',
   'TOGGLE_STEREOSCOPIC': '3d_rotation',
   'DOWNLOAD': 'download',
+  'CHAPTER': 'list',
```

Comment:
I feel like "list" would make this look too easy to confuse with the overflow menu.

I think a book-related icon could convey the idea of a chapter.
Maybe "bookmarks", "menu_book", or "library_books"?

Reply:
Given that the effect of the chapters list is it lets you go to certain parts of the presentation, "bookmarks" might be the best.
As long as we don't add an actual bookmark feature, I guess.

Reply:
Done!